tests/group_ops.rs

```diff
@@ -18,16 +18,36 @@ fn group_create_no_member() {
         false,
     ));
 
-    assert!(group_result.is_ok())
+    assert_eq!(group_result.unwrap().needs_rotation(), Some(false))
 }
 
 #[test]
 fn group_create_also_member() {
     let sdk = init_sdk();
 
     let group_result = sdk.group_create(&Default::default());
+    assert_eq!(group_result.unwrap().needs_rotation(), Some(false))
+}
+
+#[test]
+fn group_get_metadata() {
+    let admin_sdk = init_sdk();
+    let member_sdk = init_sdk();
+    let nonmember_sdk = init_sdk();
+
+    let member_id = member_sdk.device().account_id().clone();
+    let group = admin_sdk.group_create(&Default::default());
+    let group_id = &group.unwrap().id().clone();
+
+    let _ = admin_sdk.group_add_members(&group_id, &[member_id]);
```

Comment:
I don't think you need the let _ =

Reply:
I think you will get a compiler warning about an unused Result if you ignore the result...

Reply:
I see now that Craig used `?` instead. That's a good option. :+1: